prowler/lib/outputs/outputs.py

```diff
@@ -243,3 +183,63 @@ def extract_findings_statistics(findings: list) -> dict:
     stats["findings_count"] = findings_count
 
     return stats
+
+
+def add_finding_to_outputs(finding, output_options, audit_info, file_descriptors):
```

Comment:
Please include a docstring to document this function.

How can we test this new function? Please include some testing here.